### ISSUE TITLE
Can't Export 

### ISSUE THREAD

**Issue by chrisroselli** [baianat/vuse#18]
Love this project, have the Demo running locally, but cannot get the `onSave`  method to work.

 Getting these errors.
```
[Vue warn]: Property or method "onSave" is not defined on the instance but referenced during render. 
---> <App> at demo/App.vue
```
```
[Vue warn]: Invalid handler for event "saved": got undefined
---> <VuseBuilder> at src/components/VuseBuilder.vue
       <App> at demo/App.vue
```

**Comment by Abdelrahman3D** [comment]
Please show me snippets of your working code, so I can help you debugging this.

**Comment by chrisroselli** [comment]
Thanks @Abdelrahman3D 
I will get that to you ASAP!

**Comment by BeauAgst** [comment]
@Abdelrahman3D I've noticed export doesn't work in the demo either - https://baianat.github.io/vuse/example.html

**Comment by lexd** [comment]
I was able to resolve this in the demo project by adding the onSave() method into App.vue

**Comment by logaretm** [comment]
You are correct @lexd I didn't play attention when the project was being decoupled from the demo.

closed by #19 but there is another issue with how the CSS is extracted, will create another issue for it.
